R/get_fscore.R

```diff
@@ -10,13 +10,24 @@
 #'               "regression" to be consistent with
 #'               \code{\link[lavaan]{lavPredict}}, but the Bartlett scores have
 #'               more desirable properties and may be preferred for 2S-PA.
-#' @param corrected_av_efs Logical. Whether to correct for the the sampling
+#' @param corrected_fsT Logical. Whether to correct for the the sampling
 #'                         error in the factor score weights when computing
 #'                         the error variance estimates of factor scores.
+#' @param vfsLT Logical. Whether to return the covariance matrix of `fsT`
+#'              and `fsL`, which can be used as input for [vcov_corrected()]
+#'              to obtain corrected covariances and standard errors for
+#'              [tspa()] results. This is currently ignored.
 #' @param ... additional arguments passed to \code{\link[lavaan]{cfa}}. See
 #'            \code{\link[lavaan]{lavOptions}} for a complete list.
-#' @return A data frame containing the factor scores (with prefix "fs_") and
-#'         the standard errors (with suffix "_se").
+#' @return A data frame containing the factor scores (with prefix `"fs_"`),
+#'         the standard errors (with suffix `"_se"`), the implied loadings
+#'         of factor `"_by_"` factor scores, and the error variance-covariance
+#'         of the factor scores (with prefix `"evfs_"`). The following are
+#'         also returned as attributes:
+#'         * `fsT`: error covariance of factor scores
```

Comment:
Is `fsT` the new name of `vc`?

Reply:
Hi Winnie, yes I should have mentioned this earlier. I now used `fsT` for the `Theta` (error covariance) matrix of factor scores, and `fsL` for the loading matrix.